corehq/apps/export/static/export/js/models.js

```diff
@@ -56,6 +56,9 @@ hqDefine('export/js/models', function () {
             return format;
         });
 
+        self.initialSharing = ko.observable(instanceJSON.sharing);
```

Comment:
Is it deliberate that these are observables? Doesn't look like their values will ever change.

Reply:
Good point.  Will follow up with another PR (I want to get this one in for deploy)

Reply:
https://github.com/dimagi/commcare-hq/pull/22285